app/javascript/gobierto_dashboards/webapp/containers/contract/ContractsShow.vue

```diff
@@ -61,39 +66,45 @@ export default {
       initial_amount: '',
       status: '',
       process_type: '',
+      permalink: '',
       labelAsignee: I18n.t('gobierto_dashboards.dashboards.contracts.assignee'),
       labelTenderAmount: I18n.t('gobierto_dashboards.dashboards.contracts.tender_amount'),
       labelContractAmount: I18n.t('gobierto_dashboards.dashboards.contracts.contract_amount'),
       labelStatus: I18n.t('gobierto_dashboards.dashboards.contracts.status'),
-      labelProcessType: I18n.t('gobierto_dashboards.dashboards.contracts.process_type')
+      labelProcessType: I18n.t('gobierto_dashboards.dashboards.contracts.process_type'),
+      labelPermalink: I18n.t('gobierto_dashboards.dashboards.contracts.permalink')
     }
   },
   created() {
     const itemId = this.$route.params.id;
 
-    const contract = this.contractsData.find((contract) => {
-      return contract.id === itemId
+    const contract = this.contractsData.find((data) => {
```

Comment:
You might wanna save time and space 
```suggestion
    const contract = this.contractsData.find(({ id }) => id === itemId);
```